doc/source/ray-core/patterns/ray-get-loop.rst

```diff
@@ -0,0 +1,27 @@
+.. _ray-get-loop:
+
+Anti-pattern: Calling ray.get in a loop harms parallelism
+=========================================================
+
+**TLDR:** Avoid calling :ref:`ray.get() <ray-get-ref>` in a loop since it's blocking; call ``ray.get()`` only for the final result.
```

Comment:
Avoid calling :ref:`ray.get() <ray-get-ref>` in a loop since it's a blocking call; use``ray.get()`` only for the final result.